e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -154,27 +154,31 @@ describe("scenarios > embedding > smoke tests", { tags: "@OSS" }, () => {
 
           visitAndEnableSharing(object);
 
-          cy.findByTestId("embedding-settings").within(() => {
-            cy.findByRole("heading", { name: "Style" });
-            cy.findByRole("heading", { name: "Appearance" });
-            cy.findByRole("heading", { name: "Font" }).should("not.exist");
-            cy.findByRole("heading", { name: "Download data" }).should(
-              "not.exist",
+          modal().within(() => {
+            cy.findByRole("tab", { name: "Appearance" }).click();
+
+            cy.findByText("Background");
+            cy.findByText("Dashboard title");
+            cy.findByText("Border");
```

Comment:
We discourage implicit assertion. Because it's uncertain whether the author wants to assert or wants to perform actions on the element, but forgot to add assertions.

Please use `.should("exist")` for these assertions.